plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/EditConnectionDialog.java

```diff
@@ -97,6 +97,27 @@ protected Control createContents(Composite parent) {
         return contents;
     }
 
+    @Override
+    protected Point getInitialSize() {
+        Point minSize = super.getInitialSize();
+        IDialogSettings settings = getDialogBoundsSettings();
+        Point proposedSize = new Point(minSize.x, minSize.y);
+        try {
+            // Get the stored width and height.
+            int width = settings.getInt("DIALOG_WIDTH");

```

Comment:
I understand that these constants are defined in the `Dialog` class, but can you please make constants for these?